.github/workflows/continuous.yml

```diff
@@ -29,19 +29,21 @@ jobs:
         # This path is specific to Ubuntu
         path: ~/.cache/pip
         # Look to see if there is a cache hit for the corresponding requirements file
-        key: ${{ runner.os }}-pip-${{ hashFiles('cli/dev/requirements.txt') }}
+        key: ${{ runner.os }}-pip-${{ hashFiles('cli/pyproject.toml') }}
         restore-keys: |
           ${{ runner.os }}-pip-
           ${{ runner.os }}-
 
     - name: Install RACK CLI development dependencies
-      run: pip3 install -q -r cli/dev/requirements.txt
+      run: |
+        cd cli
+        pip3 install ".[dev]"
```

Comment:
I'm not familiar with this ".[dev]".  What does it mean?

Reply:
This is (kind of concerning) pip syntax that means to install the package in the current directory `.` using the optional dependency set `dev`. Another option was for me to write `rack[dev]`